examples/huggingface/docker-compose.override.yaml

```diff
@@ -0,0 +1,35 @@
+# Compose schema version
+version: '3.4'
+
+# Overriding requirements
+
+x-env: &defaults
+  GET_HOSTS_FROM: dns
+  FEDN_PACKAGE_EXTRACT_DIR: package
+  FEDN_NUM_DATA_SPLITS: 2
+
+services:
+
+  client1:
+    extends:
+      file: ${HOST_REPO_DIR:-.}/docker-compose.yaml
+      service: client
+    environment:
+      <<: *defaults
+      FEDN_DATA_PATH: /app/package/data/clients/1/enron_spam.pt
```

Comment:
think it should be /app/package/client/data... with recent changes comming from master